### PR TITLE
Mobile ads removal AB test fix

### PR DESCRIPTION
**Pull request by uplne**
Better checking for if we are inside the server side AB test and stops infinite ads for mobile.

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/create-ad-slot.js [hunk 1]
```diff
@@ -94,13 +94,16 @@ define([
         }
     };
 
-    if (config.switches.mobileTopBannerRemove && $('.top-banner-ad-container--ab-mobile').length > 0 && detect.getBreakpoint() === 'mobile') {
+    if (!!config.tests.mobileTopBannerRemove && detect.getBreakpoint() === 'mobile') {
```
Comment (kelvin-chappell) [inline static/src/javascripts/projects/common/modules/commercial/create-ad-slot.js]: What is `!!`?

Comment (uplne) [inline static/src/javascripts/projects/common/modules/commercial/create-ad-slot.js]: It is casting to a boolean. Will make non-boolean to a inverted boolean, then inverts it to get the original value as boolean. Basically this, shorter Boolean(variable) === !!variable

File: static/src/javascripts/projects/common/modules/commercial/article-body-adverts.js [hunk 2]
```diff
@@ -64,6 +64,10 @@ define([
         });
     }
 
+    function outsideMobileAdsTest() {
```
Comment (kelvin-chappell) [inline static/src/javascripts/projects/common/modules/commercial/article-body-adverts.js]: Would this be easier to read if it were `inMobileAdsTest()` 
and `if (config.switches.viewability && !inMobileAdsTest())` below?

**Comment by kelvin-chappell** [comment]
Apart from comments :+1:

File: static/src/javascripts/projects/common/modules/commercial/create-ad-slot.js (hunk 1 above)
Comment (kelvin-chappell) [inline static/src/javascripts/projects/common/modules/commercial/create-ad-slot.js]: That's quite handy.  I've never seen it before.

**Comment by heralder99** [comment]
NAVER - http://www.naver.com/
--------------------------------------------

heralder@naver.com 님께 보내신 메일 <Re: [frontend] Mobile ads removal AB test fix (#10014)> 이 다음과 같은 이유로 전송 실패했습니다.

--------------------------------------------

받는 사람이 회원님의 메일을 수신차단 하였습니다. 


--------------------------------------------

**Pull request merged**